### PR TITLE
First benchmarks

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

image

### CODE REVIEW

File: tests/docker/php/entrypoint.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-flock tests/runtime/composer-install.lock composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
+#flock tests/runtime/composer-install.lock composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
```
Comment (samdark) [inline tests/docker/php/entrypoint.sh:5]: ?

**Review by samdark — APPROVED** [review]

File: tests/docker/php/entrypoint.sh (hunk 1 above)
Comment (viktorprogger) [reply tests/docker/php/entrypoint.sh:5]: A forgotten debug

**Comment by samdark** [comment]
Should we add these to GitHub action?

**Comment by viktorprogger** [comment]
@samdark Yes, we should. I'll dive into it this week.

**Review by arogachev — APPROVED** [review]

**Comment by arogachev** [comment]
@viktorprogger Will it be done in this PR or a separate one?

**Comment by viktorprogger** [comment]
@arogachev I'm going to do it in this one.

**Pull request merged**